src/components/thumbnail/thumbnail.jsx

```diff
@@ -32,7 +33,10 @@ var Thumbnail = React.createClass({
         if (this.props.creator) {
             extra.push(
                 <div key="creator" className="thumbnail-creator">
-                    by <a href={'/users/' + this.props.creator + '/'}>{this.props.creator}</a>
+                    <FormattedMessage id={'thumbnail.by'} />{' '}
+                    <a href={'/users/' + this.props.creator + '/'}>
+                        {this.props.creator}
+                    </a>
```

Comment:
It's a temporary fix anyways – the new thumbnail mocks that we'll be using in `Explore` and `Search` can probably start to be used in our front page too.